examples/ddpg_pmsm_dq_current_control.py

```diff
@@ -20,83 +20,138 @@
 from gym.core import Wrapper
 from gym.spaces import Box, Tuple
 
+# This example shows how we can use GEM to train a reinforcement learning agent to control the current within
+# a permanent magnet synchronous motor three-phase drive

```

Comment:
I believe it would make more sense to be more precise here specialy to give all the necessary information to new users: what states are controlled in the example? 

Moreover we might just give the (trivial) information that the entire state-space is continuous.